blocks/small-promo-block/features/small-promo/default.jsx

```diff
@@ -75,51 +70,24 @@ const SmallPromoItem = ({ customFields }) => {
     }`,
   }) || null;
 
-  const imagePosition = customFields?.imagePosition || 'right';
-  const headlineMarginClass = SmallPromoStyles(imagePosition, 'headlineMargin');
-
-  const headline = customFields?.showHeadline ? (
-    <PromoHeadline
-      content={content}
-      className={headlineMarginClass}
-      linkClassName="sm-promo-headline"
-      headingClassName="sm-promo-headline"
-    />
-  ) : null;
-
-  const image = customFields?.showImage ? (
-    <div style={{ position: isAdmin ? 'relative' : null }}>
-      <div {...searchableField('imageOverrideURL')} suppressContentEditableWarning>
-        <PromoImage
-          content={content}
-          customImageURL={customFields.imageOverrideURL}
-          showPromoLabel
-          promoSize="SM"
-          imageRatio={customFields.imageRatio}
-          lazyLoad={customFields.lazyLoad}
-        />
-      </div>
-    </div>
-  ) : null;
-
   return (
-    <SmallPromoContainer
-      headline={headline}
-      image={image}
-      imagePosition={imagePosition}
+    <SmallPromoPresentation
+      content={content}
+      imageSearchField={customFields.imageOverrideURL}
```

Comment:
`imageSearchField` should be the custom field name not the value

Reply:
fixed this